src/vs/workbench/contrib/webview/browser/pre/main.js

```diff
@@ -143,7 +143,7 @@
 	function getVsCodeApiScript(allowMultipleAPIAcquire, state) {
 		const encodedState = state ? encodeURIComponent(state) : undefined;
 		return `
-			const acquireVsCodeApi = (function() {
+			globalThis.acquireVsCodeApi = (function() {
```

Comment:
@mjbvz I don't think it should cause but wanted to check with you on this change -- making it so that I can unset the global in the notebook preloads.